### ISSUE TITLE
current master travis builds fail

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/arkimet#254]
in Fedora 32 there's a compilation error:
```
../arki/types.h: In function 'std::unique_ptr<_Tp> arki::downcast(std::unique_ptr<A>)':
../arki/types.h:31:20: error: 'runtime_error' is not a member of 'std'
```
(full log: https://travis-ci.org/github/ARPA-SIMC/arkimet/jobs/753014260)

in CentOS 7 and 8 a test fails:
```
test_out_of_range (tests.test_scan_bufr.TestScanBufr) ... ../arki/runtest: line 124: 26615 Segmentation fault      (core dumped) $CMD $ARGS
Failed with result 139
```
(full log: https://travis-ci.org/github/ARPA-SIMC/arkimet/jobs/753014257)

**Comment by spanezz** [comment]
Reproduced

**Comment by brancomat** [comment]
Reopening since the compilation error in Fedora 32 is still present:
https://travis-ci.org/github/ARPA-SIMC/arkimet/jobs/754266529

**Comment by brancomat** [comment]
Fixed in f58a07d922e4e242eb063613015ee18222047491
